examples/epstein_civil_violence/civil_violence/model.py

```diff
@@ -93,12 +93,16 @@ def __init__(self, height, width, citizen_density, cop_density,
                 self.grid[y][x] = citizen
                 self.schedule.add(citizen)
 
+        # collect initial data
```

Comment:
The duplicated `collection initial data` comments seem unnecessary given that we're calling `datacollector.collect`.